### ISSUE TITLE
Reduce syntactical nature of clues

### ISSUE THREAD

**Issue by simonlindholm** [Gullesnuffs/Codenames#13]
E.g. verbs are currently similar to other verbs, which is natural, but not what we prefer for codenames clues. Maybe one can do some smart linear algebra thing??

**Comment by HalfVoxel** [comment]
Do we know which words are verbs at the moment?

**Comment by simonlindholm** [comment]
No. And of course, many of the standard wordlist words are both verbs and nouns. I think the latest example I had was ROSE being hinted at with another past tense verb.

**Comment by simonlindholm** [comment]
ROSE, not SHOT. anyway.
